README.md

```diff
@@ -0,0 +1,17 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Learning Linting, HTML and CSS.
+
+
+## Built With
+
+- HTML, CSS, Javascript
+
+
```

Comment:
> Please, kindly refer to the [microverse readme template](https://github.com/microverseinc/readme-template) to add the missing headers in your readme and add the content relevant to your current project to suit your needs.
- Add the "Getting Started" header section and add some instructions on how users can have your project running locally on their machine.
- Add the Author section with the appropriate links to your social media platforms. 
- Add the "Contributing" section and its contents ensuring the issues page references that of this current project on Github.
- Add the "Acknowledgments" section and its contents.

Reply:
*LINTER*

- You have done well adding the linter file, however, you've not added the `.hintrc` and `stylelintrc.json` linter files. Please kindly refer to the [html-css](https://github.com/microverseinc/linters-config/tree/master/html-css) to add the remaining and ensure there are no linters error before you submit for a re-review.

Reply:
*PR DESCRIPTION*

- Please, kindly add a description in your PR, summarizing what you've done. check the image below to see how to add one:

*Description*
![des](https://user-images.githubusercontent.com/59397159/184932430-84152e54-a76b-418c-961a-5d7464fa469f.png)

*PR TITLE*

- Please, kindly rename your PR's Title to "Hello Microverse Project" as this is more descriptive compare to `Linter for html` as you've added.